rails-commenting.rb

```diff
@@ -39,7 +39,7 @@ def edit
 
   def update
     @post = BlogPost.find(params[:id])
-    # 6)
+    # 6) @post is made to update(which routes to put and patch) a post while still using the security of strong params to keep hackers out.
```

Comment:
This is using the .update method and taking in the private method as an argument to pass in the params for the update to actually update in the database.